1-2/main.js

```diff
@@ -0,0 +1,15 @@
+function quiz() {
+  let answer = true;
+  
+  while (answer) {
+    const reply = prompt('日本の首都は？');
+    if(reply !== '東京'){
+      alert('不正解です!');
+    }else if(reply == '東京'){
```

Comment:
`===`を使用してより厳密な比較を行いましょう。
```suggestion
    } else if (reply === '東京'){
```